recipe/meta.yaml

```diff
@@ -14,21 +14,20 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://files.pythonhosted.org/packages/source/b/beakerx/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  
+
 build:
-  number: 3
-  skip: true  # [py2k]
-  skip: true  # [win32]
```

Comment:
This was duplicated and was causing some errors. We can safely remove those anyway as they are not relevant anymore.